### PR TITLE
Circos legend/readme update

### PR DESCRIPTION
**Pull request by walter-hernandez**
…of nodes for the circus plot

This PR resolves issue #_______.

# PR Checklist

Please ensure that you have done the following:

1. [✓] PR in from a fork off your branch. Do not PR from <your_username>:master, but rather from <your_username>:<branch_name>.
2. [✓] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

## Code Changes

If you are adding code changes, please ensure the following:

- [✓] Ensure that you have added tests.
- [✓] Run all tests (`$ pytest .`) locally on your machine.
    - [✓] Check to ensure that test coverage covers the lines of code that you have added.
    - [✗] Ensure that all tests pass:
       - The test is not passed because the baseline image does not include a legend since this is a new feature added.

## Documentation Changes

If you are adding documentation changes, please ensure the following:

- [✓] Build the docs locally.
- [✓] View the docs to check that it renders correctly.

# PR Description

Please describe the changes proposed in the pull request:

- Added the argument "group_legend" for the user to indicate if it wants the graph to include the legend or not. 
- Exposed the legend_handles in the case that the user wants to customize the legend

Included dummy examples in the directory "examples\circos"for:

- The default legend:

circusplot_defaultlegend

- How to customize the legend using the CircosPlot().legend_handles

circusplot_customlegend

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

File: examples/circos/group_labels_with_custom_legend.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    fontsize=10,
+    group_legend=True,
+    figsize=(12, 12)
+)
+
+# Draw the CircosPlot
+c.draw()
+c.figure.tight_layout()
+
+# If you want more control over the legend, you can pass c.legend_handles to plt.legend()
```
Comment (ericmjl) [inline examples/circos/group_labels_with_custom_legend.py]: Wonderful example!

**Comment by ericmjl** [comment]
@walter-hernandez this is a very well-done PR! I'm very impressed at the code quality.

It looks like the PR is only failing on codestyle. Could you fix those by any chance? A convenience command, `make check`, should do the trick. (It will execute all code formatting _and_ re-run all tests.)

Other than that, once you've pushed up those changes, I have no problems merging!

**Comment by walter-hernandez** [comment]
Corrected the code style and verified that it passed the pycodestyle

**Comment by ericmjl** [comment]
Apologies for letting this hang, I've had evening engagements after work that I had to attend to. Ready to merge, will be cutting a new release soon!

**Pull request merged**